test/parallel/test-crypto-dh.js

```diff
@@ -22,24 +22,29 @@ assert.strictEqual(secret2.toString('base64'), secret1);
 assert.strictEqual(dh1.verifyError, 0);
 assert.strictEqual(dh2.verifyError, 0);
 
-const argumentsError =
-  /^TypeError: First argument should be number, string, Buffer, TypedArray, or DataView$/;
-
-assert.throws(() => {
+common.expectsError(function() {
```

Comment:
This is obviously good, but you can notice that the RegExp included also the Type (`/^TypeError: .../`).
So you should add
```js
  type: TypeError
```
to all the assertion below.